packages/api/src/types.ts

```diff
@@ -110,6 +110,7 @@ interface StorageEntryPromiseMulti {
 export interface StorageEntryBase<ApiType extends ApiTypes, F extends AnyFunction> {
   at: (hash: Hash | Uint8Array | string, ...args: Parameters<F>) => PromiseOrObs<ApiType, ObsInnerType<ReturnType<F>>>;
   creator: StorageEntry;
+  entries: () => PromiseOrObs<ApiType, [StorageKey, ObsInnerType<ReturnType<F>>][]>;
```

Comment:
@amaurymartiny Can you check this typing? Not sure it is valid.

Reply:
LGTM